### PR TITLE
Responsiveness

### PR DESCRIPTION
**Pull request by jasonrees1**
CSS and HTML changes to add responsiveness for up to 640px

### CODE REVIEW

File: jrportfolio.css [hunk 1]
```diff
@@ -152,8 +153,9 @@ html {
 .box-1container {
 	float: left;
 	width: 33.3%;
-	height: 340px;
+	height: auto;
 	padding: 0 50px;
+	margin-bottom:0.5rem;
```
Comment (ashleycoles) [inline jrportfolio.css]: Make sure there is a space between the property and value on all rules, eg: margin-bottom: 0.5rem;

File: jrportfolio.css [hunk 2]
```diff
@@ -245,4 +247,103 @@ html {
...
+	.name p {
+		font-size: 1.3rem;
+		margin-top: 10px;
+	}
+
+	.rightlink {
+		float: right;
+		width: 40%;
+		text-align: right;
+		margin-right:10%;
```
Comment (ashleycoles) [inline jrportfolio.css]: Space between property and value.

File: jrportfolio.css [hunk 3]
```diff
@@ -245,4 +247,103 @@ html {
...
+		font-size: 1.5rem;
+		color: #000;
+	}
+
+	.headercontainer a:hover {
+		color: #7e8187;
+	}
+
+	.headercontainer .leftlink,.headercontainer .rightlink {
+		margin-top:2rem;
```
Comment (ashleycoles) [inline jrportfolio.css]: Space between property and value.

**Review by ashleycoles — CHANGES_REQUESTED** [review]

File: jrportfolio.css (hunk 1 above)
Comment (jasonrees1) [reply jrportfolio.css]: Done

File: jrportfolio.css (hunk 2 above)
Comment (jasonrees1) [reply jrportfolio.css]: Done

File: jrportfolio.css (hunk 3 above)
Comment (jasonrees1) [reply jrportfolio.css]: Done

**Review by ashleycoles — APPROVED** [review]

**Pull request merged**